airflow-core/src/airflow/ui/src/i18n/locales/he/common.json

```diff
@@ -0,0 +1,79 @@
+{
+  "defaultToGraphView": "ברירת מחדל תצוגת גרף",
+  "defaultToGridView": "ברירת מחדל תצוגת רשת",
+  "logout": "התנתק",
+  "switchToDarkMode": "החלף למצב כהה",
+  "switchToLightMode": "החלף למצב בהיר",
+  "timezone": "איזור זמן",
+  "user": "משתמש",
+  "language": {
+    "chinese": "סינית מסורתית",
+    "english": "אנגלית",
+    "german": "גרמנית",
+    "korean": "קוריאנית",
+    "select": "בחר"
+  },
+  "nav": {
+    "home": "מסך הבית",
+    "assets": "משאבים (Assets)",
+    "browse": "עיין",
+    "admin": "אדמין",
+    "docs": "דוקומנטציה",
+    "plugins": "פלאגינים"
+  },
+  "browse": {
+    "auditLog": "לוג מערכת",
+    "xcoms": "(XComs)"
```

Comment:
Why the parentheses?

Reply:
The original one was translation for Xcom and then the (xcom) is to further explain.
didn't know how to quite translate this one yet

Reply:
רעיון ראשוני: חבילות פלט

Reply:
I think that it's best to keep this term in English, rather than translate or transliterate it. Its original meaning is "Cross-communications" - I don't think that `תקשור(ו)ת צולב(ו)ת` or `אקסקום` would do any better.